packages/docs/docusaurus.config.js

```diff
@@ -7,8 +7,8 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   title: 'AI.JSX',
-  tagline: 'Dinosaurs are cool',
-  favicon: 'img/favicon.ico',
+  tagline: 'The Toolkit for building Responsive AI Apps',
```

Comment:
let's not get into the product positioning thing here.
```suggestion
  tagline: 'The Toolkit for building JSX AI Apps',
```